uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/ContactTest.php

```diff
@@ -83,21 +83,53 @@ public function testGetLeaveManageesDoesNotReturnFilteredOutFieldsWhenFilteredOu
     $this->assertArrayNotHasKey('modified_date', $result['values'][$this->contact2['id']]);
   }
 
-  public function testGetLeaveManageesOnlyReturnsContactsManagedByTheCurrentLoggedInManager() {
+  public function testGetLeaveManageesOnlyReturnsContactsManagedByTheContactPassedInManagedByParameter() {
     $contact3 = ContactFabricator::fabricate();
     $manager2 = ContactFabricator::fabricate();
 
     $this->setContactAsLeaveApproverOf($this->manager, $this->contact2, null, null, true, 'has things managed by');
     $this->setContactAsLeaveApproverOf($this->manager, $this->contact1, null, null, true, 'has leaves approved by');
     $this->setContactAsLeaveApproverOf($manager2, $contact3, null, null, true, 'has leaves approved by');
 
-    $result = civicrm_api3('Contact', 'getleavemanagees');
+    $result = civicrm_api3('Contact', 'getleavemanagees', ['managed_by' => $this->manager['id']]);
 
-    //only the two contacts who are managed by the current logged in manager are returned
     $this->assertEquals(2, $result['count']);
     $this->assertEquals($result['values'][$this->contact1['id']]['id'], $this->contact1['id']);
     $this->assertEquals($result['values'][$this->contact1['id']]['display_name'], $this->contact1['display_name']);
     $this->assertEquals($result['values'][$this->contact2['id']]['id'], $this->contact2['id']);
     $this->assertEquals($result['values'][$this->contact2['id']]['display_name'], $this->contact2['display_name']);
   }
```

Comment:
can you also add a security test? If I'am logged in as manager 1 but then I call the API passing the manager 2 ID I should get an empty response (Because I should not have permission to see other manager's managees)